packages/rocketchat-ui-flextab/flex-tab/flexTabBar.js

```diff
@@ -0,0 +1,50 @@
+Template.flexTabBar.helpers({
+	active() {
+		if (this.template === Template.instance().tabBar.getTemplate() && Template.instance().tabBar.getState() === 'opened') {
+			return 'active';
+		}
+	},
+
+	buttons() {
+		return RocketChat.TabBar.getButtons();
+	},
+
+	title() {
+		return t(this.i18nTitle) || this.title;
+	},
+
+	visible() {
+		if (this.groups.indexOf(Template.instance().tabBar.currentGroup()) === -1) {
+			return 'hidden';
+		}
+	},
+
+	opened() {
+		return Template.instance().tabBar.getState();
+	},
+
+	template() {
+		return Template.instance().tabBar.getTemplate();
+	},
+
+	flexData() {
+		return Object.assign(Template.currentData().data || {}, {
+			tabBar: Template.instance().tabBar
+		});
+	}
+});
+
+Template.flexTabBar.events({
+	'click .tab-button': function(e, instance) {
```

Comment:
Can you replace `'click .tab-button': function(e, instance) {` by `'click .tab-button'(e, instance) {`?